collectors/persistentvolumeclaim.go

```diff
@@ -87,9 +98,25 @@ func (collector *persistentVolumeClaimCollector) Collect(ch chan<- prometheus.Me
 	}
 }
 
+// getPersistentVolumeClaimClass returns StorageClassName. If no storage class was
+// requested, it returns "".
+func getPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
+	// Use beta annotation first
+	if class, found := claim.Annotations[v1.BetaStorageClassAnnotation]; found {
+		return class
+	}
+
+	if claim.Spec.StorageClassName != nil {
+		return *claim.Spec.StorageClassName
+	}
+
+	return ""
```

Comment:
@cofyc According to https://github.com/kubernetes/kube-state-metrics/blob/master/collectors/pod.go#L257-L266, pod uses `<none>` as the default value for none exist label values.

> maybe we need to document it somewhere, for developers and also for end-users.

👍 . Absolutely we should do this.